src/character.cpp

```diff
@@ -5683,31 +5683,32 @@ void Character::try_reduce_weariness( const float exertion )
     weary.tick_counter++;
     if( exertion == NO_EXERCISE ) {
         weary.low_activity_ticks++;
-        // Recover twice as fast at rest
+        // Recover twice as fast if asleep/similar
         if( in_sleep_state() ) {
             weary.low_activity_ticks++;
         }
     }
 
     const float recovery_mult = get_option<float>( "WEARY_RECOVERY_MULT" );
 
-    if( weary.low_activity_ticks >= 6 ) {
+    if( weary.low_activity_ticks >= 1 ) {
         int reduction = weary.tracker;
         const int bmr = base_bmr();
-        // 1/20 of whichever's bigger
+        // 1/120 of whichever's bigger
         if( bmr > reduction ) {
-            reduction = bmr * recovery_mult;
+            reduction = std::floor( bmr * recovery_mult * weary.low_activity_ticks / 6 );
```

Comment:
We should make sure that floating point division is happening here, not integer division.
This will ensure it happens:
```suggestion
            reduction = std::floor( bmr * recovery_mult * weary.low_activity_ticks / 6.0f );
```

Reply:
Thank you, I'm still a bit uncertain about float vs integer in C++ (it's rather different than in, say, Python).

Reply:
Yeah, I'm not quite sure on the rules either - this may have been fine, or it may have needed adjustment (or worse, it may have been fine on some platforms, and not others), so let's be safe.